RelayProofsOfConcept/EddsaTokioServer/protocols.json

```diff
@@ -3,7 +3,7 @@
     {
       "id": 1,
       "names": ["Multi-party-eddsa","multi-party-eddsa", "multi_party_ed25519"],
-      "capacities": [1, 2, 3, 4, 5]
+      "capacities": [1, 2, 3, 4, 5, 10]
```

Comment:
what does it means?

Reply:
The number of participants allowed to partake in the protocol.